File: appserver-legacy/src/main/java/org/radarbase/appserver/service/transmitter/DataMessageTransmitter.java
package org.radarbase.appserver.service.transmitter;

import org.radarbase.appserver.entity.DataMessage;
import org.radarbase.appserver.exception.MessageTransmitException;

public interface DataMessageTransmitter {
    void send(DataMessage dataMessage) throws MessageTransmitException;
}

File: appserver-jersey/src/main/kotlin/org/radarbase/appserver/jersey/service/transmitter/DataMessageTransmitter.kt
package org.radarbase.appserver.jersey.service.transmitter

import org.radarbase.appserver.jersey.entity.DataMessage
import org.radarbase.appserver.jersey.exception.MessageTransmitException

interface DataMessageTransmitter {
    @Throws(MessageTransmitException::class)
    fun send(dataMessage: DataMessage)
}
